db/migrate/20230815235448_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author, foreign_key: { to_table: :users }
+      t.integer :post, foreign_key: { to_table: :posts }
```

Comment:
- [ ] It would be nice if you put `t.reference` instead of `t.integer` since it is a foreign key 🤔 to make your migrations as in the project requirements 👍 

NOTE: Kindly make the above change because I am not able to see any reference of comments with the post table in the schema.rb file due to t.integer instead of t.reference 🤔